apps/archive/archive_media.py

```diff
@@ -44,13 +43,11 @@ def on_create(self, docs):
             content_type = magic.from_buffer(doc['media'].read(1024), mime=True)
             doc['media'].seek(0)
             file_type = content_type.split('/')[0]
-            if file_type == 'video' and app.config.get("VIDEO_SERVER_ENABLE"):
-                if not self.videoEditor.check_video_server():
-                    raise SuperdeskApiError(message="Cannot connect to videoserver", status_code=500)
+            if file_type == 'video' and app.config.get("VIDEO_SERVER_ENABLED"):
```

Comment:
Should update this on `default_settings` too

Reply:
hi @idsvn , did not add all files to a commit